src/components/NpsSurvey.js

```diff
@@ -46,10 +46,16 @@ export const NpsSurvey = _.flow(
   }
 
   async loadStatus() {
-    const lastResponse = (await Ajax().User.lastNpsResponse()).timestamp
-    const oneMonthAgo = _.tap(d => d.setDate(d.getMonth() - 1), new Date())
+    const lastResponse = await Ajax().User.lastNpsResponse()
 
-    this.setState({ requestable: !lastResponse || (new Date(lastResponse) < oneMonthAgo) })
+    const currTime = new Date()
+    const currTimeHours = Date.parse(currTime)/3600000
+
+    const askTheUser = lastResponse
+      ? currTimeHours - new Date(lastResponse).getTime()/3600000 >= 168
+      : currTimeHours - Date.parse(await Ajax().User.firstTimestamp(currTime))/3600000 >= 24
```

Comment:
This area is a little tough to understand what's going on. Maybe pulling things out and naming them can help with readability (especially for future devs looking at this code)?